src/status.rs

```diff
@@ -79,11 +79,11 @@ pub struct RadioShow {
 }
 
 /// The main Space API status object.
-#[derive(Debug, Clone)]
+#[derive(Debug, Clone, RustcDecodable)]
 pub struct Status {
 
     // Hackerspace properties
-    pub api: &'static str,
```

Comment:
Ah, that's a very good point :) We just need to remember that we have to bump the minor version due to an incompatibility. Reminds me that we should introduce a changelog.